src/app/add-post/add-post.component.scss

```diff
@@ -0,0 +1,10 @@
+.form-group {
```

Comment:
We want to move to Angular Material but I have to say that Bootstrap does not need to be styled like this. Everything that this rule declares can be applied with the classes `d-flex`, `align-center` and `p-3` (or similar).

Modifying Boostrap classes is not advised anyway, even at component-level.